### DISCUSSION TITLE
Migrated the TinyMCE from 4.5.4 to 7.3.0 but Font and Foont Size selection is not displayed in Rich text editor

### DISCUSSION THREAD

**Discussion by Rajanusa** [tinymce/tinymce#9843] [General]
The .Net Framework 4.7.1 MVC Web App use TinyMCE. 
Migrated the TinyMCE from 4.5.4 to 7.3.0 but Font and Foont Size selection is not displayed in Rich text editor.
Please advise

**Comment by lostkeys** [comment]
Hi @Rajanusa! Have you double checked the toolbar button names in your config with the latest documentation?
